allensdk/brain_observatory/behavior/vbo_2023_nwb_writers/ophys/schemas.py

```diff
@@ -0,0 +1,151 @@
+import marshmallow as mm
+import pandas as pd
+from allensdk.brain_observatory.argschema_utilities import (
+    RaisingSchema,
+    check_write_access_overwrite,
+)
+from allensdk.brain_observatory.behavior.vbo_2023_nwb_writers.behavior.schemas import (  # noqa: E501
+    BaseInputSchema,
+    BehaviorSessionMetadataSchema,
+)
+from allensdk.core.utilities import literal_col_eval
+from argschema.fields import Int, Nested, String
+
+COL_EVAL_LIST = ["ophys_experiment_id", "ophys_container_id", "driver_line"]
+
+
+class OphysExperimentMetadataSchema(BehaviorSessionMetadataSchema):
```

Comment:
These schemas are more related to the project-level metadata tables than writing nwb files. For better or worse, we have metadata table logic in both `allensdk/brain_observatory/behavior/behavior_project_cache/tables` for vbo and `allensdk/brain_observatory/vbn_2022/metadata_writer` for vbn. I think these schemas should be placed closer to logic for the tables rather than writing nwb files. Possibly in `allensdk/brain_observatory/behavior/behavior_project_cache/tables`